UIKitDZ/UIKitDZ/Choose Coffee screen/ChooseCoffeeViewController.swift

```diff
@@ -1,18 +1,120 @@
+    }
+
+    private func addViews() {
+        view.addSubview(chooseView.uiView)
+        view.addSubview(chooseView.imageCoffee)
+        view.addSubview(chooseView.segmentControl)
+        view.addSubview(chooseView.modificationLabel)
+        view.addSubview(chooseView.coffeeBeansButton)
+        view.addSubview(chooseView.beansImage)
+        view.addSubview(chooseView.beansLabel)
+        view.addSubview(chooseView.ingredientsButton)
+        view.addSubview(chooseView.ingredientsLabel)
+        view.addSubview(chooseView.pluseImage)
+        view.addSubview(chooseView.priceLabel)
+        view.addSubview(chooseView.enterButton)
+    }
+
+    // Меняем картинки в сегмент контроллере
+    @objc private func selectedValue(sender: UISegmentedControl) {
+        if sender == chooseView.segmentControl {
+            let segmentIndex = sender.selectedSegmentIndex
+            chooseView.imageCoffee.image = segmentImages[segmentIndex]
+        }
+    }
+
+    // Возвращаеися на прошлый vc
+    @objc private func goBackViewController() {
+        let menuController = MenuViewController()
+        navigationController?.popToViewController(menuController, animated: true)
+    }
+
+    // Переход на экран с зернами
+    @objc private func gotoCoffeeBeansScreen() {
+        let coffeeBeansController = CoffeeBeansController()
+        present(coffeeBeansController, animated: true)
+    }
+
+    // Переход на экран ингредиентов и перенос цены в Order Screen
+    @objc private func goToIngredientsScreen() {
+        let ingredientsController = IngredientsViewController()
+        ingredientsController.summaUp = { newOrder in
+            self.chooseView.priceLabel.text = newOrder
+            if newOrder == "Цѣна - 100 руб" {
+                self.chooseView.pluseImage.image = UIImage(named: "pluse")
+            } else {
+                self.chooseView.pluseImage.image = UIImage(named: "checkMark")
+            }
+        }
+        present(ingredientsController, animated: true)
+    }
+
+    // При нажатии на кнопку с переходом в Order у нас при закрытие ордера вызывается делегат с помощью которого пушится
+    // Sms screen
+    @objc private func goToOrderScreen() {
+        let orderController = OrderViewController()
+        orderController.push = {
+            let smsController = SmsViewController()
+            self.navigationController?.pushViewController(smsController, animated: true)
+        }
+        orderController.orderView.priceLabel.text = chooseView.priceLabel.text
+        present(orderController, animated: true)
+    }
+
+    // Метод что б поделиться смс
+    @objc private func shareToTelegram() {
+        let message = "Лови промокод roadmaplove на любой напиток из Кофейнов"
+        let activityViewController = UIActivityViewController(activityItems: [message], applicationActivities: nil)
+        present(activityViewController, animated: true)
+    }
+
+    // MARK: - Constants
```

Comment:
```suggestion
    // MARK: - Constants
```
на верху в enum только